administrator/language/de-DE/de-DE.com_joomlaupdate.ini

```diff
@@ -34,7 +34,11 @@ COM_JOOMLAUPDATE_VIEW_COMPLETE_HEADING="Joomla!-Versionsaktualisierungsstatus"
 COM_JOOMLAUPDATE_VIEW_COMPLETE_MESSAGE="Die Site wurde aktualisiert. Die neue Joomla!-Version ist jetzt: %s."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DOWNLOAD_IN_PROGRESS="Aktualisierung wird heruntergeladen. Bitte warten ..."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_DOWNLOAD_URL="Es konnte keine Download-URL gefunden werden"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_DOWNLOAD_URL_DESC="Eine Aktualisierung für Joomla! %1$s wurde gefunden, allerdings konnte die Download-URL nicht ermittelt werden. Hierzu kann es zwei Gründe geben:<br /><ul><li>Die Webserverumgebung unterstützt die <a href=\"https://www.joomla.org/technical-requirements.html\" target=\"_blank\" rel=\noopener noreferrer\">minimalen Systemanforderungen für Joomla! %1$s (engl.)</a> nicht und es gibt keinen alternativen Download für diese Umgebung.</li><li>Es gibt ein Problem mit den Joomla!-Aktualisierungsservern.</li></ul><br />Bitte das Aktualisierungspaket von der <a href=\"https://www.joomla.org/download.html\" target=\"_blank\" rel=\noopener noreferrer\">offiziellen Joomla!-Downloadsite (engl.)</a> herunterladen und anschließend die Funktion „Hochladen & Aktualisieren“ nutzen."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NO_DOWNLOAD_URL_DESC="Eine Aktualisierung für Joomla! %1$s wurde gefunden, allerdings konnte die Download-URL nicht ermittelt werden. Bitte das Aktualisierungspaket von der <a href=\"https://downloads.joomla.org/latest\" target=\"_blank\" rel=\"noopener noreferrer\">offiziellen Joomla!-Downloadsite (engl.)</a> herunterladen und anschließend die Funktion „Hochladen & Aktualisieren“ nutzen."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_DB_NOT_SUPPORTED="Datenbanktyp wird nicht unterstützt"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_DB_NOT_SUPPORTED_DESC="Ein Aktualisierung für Joomla! %1$s wurde gefunden, allerdings wird der aktuelle Datenbanktyp nicht unterstützt oder entspricht nicht <a href=\"https://downloads.joomla.org/technical-requirements\">den Mindestanforderungen für Joomla %1$s</a>."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_PHP_VERSION_NOT_SUPPORTED="PHP-Version wird nicht unterstützt"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_PHP_VERSION_NOT_SUPPORTED_DESC="Eine Aktualisierung für Joomla! %1$s wurde gefunden, allerdings entspricht die aktuell installierte PHP-Version nicht <a href=\"https://downloads.joomla.org/technical-requirements\">den Mindestanforderungen für Joomla %1$s</a>."
```

Comment:
in 39 schreiben wir wird nicht unterstützt oder entspricht nicht, sollen wir das hier auch vereinheitlichen @zero-24